mkdocs2sphinx/convert_files.py

```diff
@@ -99,28 +96,30 @@ def convert_files(path, block_list, replacement_map, license_text, files_no_lice
     """
     stats = defaultdict(int)
 
-    for root, dirs, files in os.walk(path):  # pylint: disable=unused-variable
-        for fl in files:
-            # get the file extension
-            _, ext = os.path.splitext(fl)
-            ext = ext.lower()
-
-            # only converting HTML files at the moment
-            if ext not in {".html", ".css", ".js"} and not ext.endswith("_t"):
-                continue
-
-            # we need to read and then write back to this file
-            with io.open(os.path.join(root, fl), "r+", encoding="utf-8") as write_file:
-                file_contents = write_file.read()
-                file_contents = remove_blocks(file_contents, block_list)
-                file_contents = do_replacements(file_contents, replacement_map, stats)
-
-                if fl not in files_no_license:
-                    file_contents = prepend_license(license_text, file_contents, fl)
-
-                # reset to start of file, write, and then truncate
-                write_file.seek(0)
-                write_file.write(file_contents)
-                write_file.truncate()
+    for fl in path.glob("**/*"):
```

Comment:
Actually you can also do:
```suggestion
    for fl in path.rglob("*"):
```
and save 2 more characters here !

Reply:
depends if you want the full recursion or not. 
`*` give the first level only

Reply:
It's `rglob` here: https://docs.python.org/3/library/pathlib.html#pathlib.Path.rglob so it should be equivalent

Reply:
ahh I missed the `r`glob ! 
indeed !